media-libs/libsidplay/files/libsidplay-2.1.1-autoconf.patch

```diff
@@ -0,0 +1,513 @@
+---with-sidplay2-includes=$pwd/libsidplay/include \
+---with-sidplay2-library=$pwd/libsidplay/src \
+---with-resid-includes=$pwd/resid \
+---with-resid-library=$pwd/resid \
+---libdir=$builders;
+-then
+-exit
+-fi
++AX_SUBDIRS_CONFIGURE([builders/resid-builder], [
++        [--disable-library-checks],
++        [--with-sidplay2-includes=$pwd/libsidplay/include],
++        [--with-sidplay2-library=$pwd/libsidplay/src],
++        [--with-resid-includes=$pwd/resid],
++        [--with-resid-library=$pwd/resid],
++        [--libdir=$builders]])
+ 
+-if test x$hardsid = xtrue; then
+-    echo; echo "Configuring hardsid-builder..."
+-    cd $pwd/builders/hardsid-builder
+-    if ! eval ./configure $ac_configure_args \
+-        --with-sidplay2-includes=$pwd/libsidplay/include \
+-        --with-sidplay2-library=$pwd/libsidplay/src \
+-        --libdir=$builders;
+-    then
+-    exit
+-    fi
+-fi
++AM_CONDITIONAL(HARDSID, test x$hardsid = xtrue)
+ 
+-echo; echo "Configuring libsidutils..."
+-cd $pwd/libsidutils
+-if ! eval ./configure $ac_configure_args \
+---with-sidplay2-includes=$pwd/libsidplay/include \
+---with-sidplay2-library=$pwd/libsidplay/src;
+-then
+-exit
++if test x$hardsid = xtrue; then
++    AX_SUBDIRS_CONFIGURE([builders/hardsid-builder], [
++            [--disable-library-checks],
++            [--with-sidplay2-includes=$pwd/libsidplay/include],
++            [--with-sidplay2-library=$pwd/libsidplay/src],
++            [--libdir=$builders]])
+ fi
+ 
+-echo
+-cd $pwd
++AX_SUBDIRS_CONFIGURE([libsidutils], [
++        [--disable-library-checks],
++        [--with-sidplay2-includes=$pwd/libsidplay/include],
++        [--with-sidplay2-library=$pwd/libsidplay/src]])
+ 
+ AC_OUTPUT(
+ Makefile \
+diff --git a/libsidplay/configure.ac b/libsidplay/configure.ac
+index 1946508..9c9d808 100644
+--- a/libsidplay/configure.ac
++++ b/libsidplay/configure.ac
+@@ -8,6 +8,7 @@ AC_INIT(libsidplay,2.LIBCUR.LIBREV)
+ AC_CONFIG_HEADER(unix/config.h)
+ AC_CONFIG_SRCDIR(Makefile.am)
+ AC_CONFIG_AUX_DIR(unix)
++AC_CONFIG_MACRO_DIR([unix])
+ AM_INIT_AUTOMAKE(no-define)
+ 
+ dnl libtool-style version-info number
+diff --git a/libsidutils/configure.ac b/libsidutils/configure.ac
+index 1e38d14..4461aa1 100644
+--- a/libsidutils/configure.ac
++++ b/libsidutils/configure.ac
+@@ -2,6 +2,7 @@ dnl Process this file with autoconf to produce a configure script.
+ AC_INIT(Makefile.am)
+ AC_CONFIG_AUX_DIR(unix)
+ AM_CONFIG_HEADER(unix/config.h)
++AC_CONFIG_MACRO_DIR([unix])
+ 
+ dnl Setup library CURRENT, REVISION and AGE
+ LIBCUR=0
+diff --git a/resid/configure.in b/resid/configure.ac
+similarity index 100%
+rename from resid/configure.in
+rename to resid/configure.ac
+diff --git a/unix/ax_subdirs_configure.m4 b/unix/ax_subdirs_configure.m4
+new file mode 100644
+index 0000000..01cb074
+--- /dev/null
++++ b/unix/ax_subdirs_configure.m4
+@@ -0,0 +1,334 @@
++# ===========================================================================
```

Comment:
Could you drop this bit and just BDEPEND on autoconf-archive?

Reply:
Sure, done.